### ISSUE TITLE
Testing the problem of loss oscillation

### ISSUE THREAD

**Issue by rongyua** [FT-ZHOU-ZZZ/CMTA#3]
Hello, thank you very much for your contribution to the paper. However, when I reproduced the paper, I found that the training loss has been declining. The test loss dropped to the lowest point after running for several epochs and then began to rise and fall again. And sometimes best_test_c is not testing. Obtained when the loss is lowest, is this a normal phenomenon? I would be grateful if you could answer it.

**Comment by rongyua** [comment]
Sorry, there was a problem with the translation of the first question, please ignore it.
Hello, thank you very much for your contribution to this article. However, when I reproduced the code, I found that the training loss kept going down. The test loss bottoms out after running for a few epochs and then starts to rise and fall again. And sometimes when test_loss is the lowest, the best_test_c index cannot be obtained. Is this normal? I would be grateful if you could answer.

**Comment by FT-ZHOU-ZZZ** [comment]
Sorry for late reply.
The case you mentioned is very common.
There is no absolutely positive correlation between overall loss and c-index.
Please check the implementation of nll_surv loss and calculation of c-index.
